dbt/models/default/docs.md

```diff
@@ -55,22 +65,50 @@ workbooks rather than iasWorld.
 
 ### Assumptions
 
-* `cur = 'Y'` or a single row for a parcel even when `cur = 'N'` indicate rows
-that should be included from `oby` and `comdat`
-* Null proration rate for condo unit indicates condo isn't associated with other
-units
-* Proration rates in `oby` and `comdat` are parcel-level
-* `effyr` is equivalent to `yrblt` when `yrblt` is `NULL`
-* Most recent value for CDU is most relevant if it has been re-coded to `NULL`
+- `cur = 'Y'` or a single row for a parcel even when `cur = 'N'` indicate rows
```

Comment:
I think we have cur = 'Y' applied to pardat, comdat, and oby when building this view, i don't think cur = 'N' can make it in

Reply:
Removed in 268e53563c2fcca2d0e397eaad6b4c2296345daa!